src/Simple/Evaluator.php

```diff
@@ -63,6 +63,9 @@ private function execute($callable, $arguments)
             if ($e instanceof JsonRpc\Exception) {
                 throw $e;
             } else {
+                // Send the original exception to the error log (useful for debugging purposes)
+                error_log($e);
```

Comment:
Seems reasonable. I'll test to make sure that works later today and update the pull request.

Thanks